scripts/output.js

```diff
@@ -122,7 +122,7 @@ function generate(content, opt)
   <span id="data" data-root="/" data-search="/${tree}/search" data-tree="${tree}"></span>
   <span id="state" data-offset="${stateOffset}" data-limit="${stateLimit}" data-result-count="${resultCount}" data-eof="${eof}"></span>
 
-  <script src="/static/js/dxr.js"></script>
+  <script src="/static/js/search.js"></script>
```

Comment:
Note that when this merges we'll likely need to retrigger the config3.json indexer (manually) since that will have generated files still linking to `dxr.js`